app/client/src/features/events/EventSidebar/EventSidebar.tsx

```diff
@@ -90,7 +90,7 @@ export interface EventSidebarProps {
 }
 export const EventSidebar = ({ fragmentRef }: EventSidebarProps) => {
```

Comment:
Should these changes be in a different PR?  They don't seem login/logout related?

Reply:
Yeah it probably should be moved to a general bugfix branch.